### ISSUE TITLE
Is this action still supported?

### ISSUE THREAD

**Issue by buckett** [einaregilsson/beanstalk-deploy#93]
Hiya @einaregilsson

Are you still looking to support this project? I ask because there doesn't seem to have been any activity in this repository from you since the end of 2021. In the near future this action will need to be updated to address the end of support for Node 12 (https://github.com/einaregilsson/beanstalk-deploy/pull/92) and I thought I'd get the discussion started before is become critical.

This project has been really useful for us and if there's some help we could provide do shout.

**Comment by buckett** [comment]
I'm really sorry to report this but it looks like Einar passed away earlier this year:

https://www.facebook.com/cardgames.io/posts/pfbid0V3Wqq6Ze5whVEPDzXR1oKbru2khntRioXKFQwXPkeBEVq9XrrULqanee2qmzWTY9l

**Comment by gzur** [comment]
> Einar passed away earlier this year

You are correct. My name is Gissur and I have assumed stewardship over Einar's GitHub account on behalf of his widow.

I'll do my best to keep the lights on and then we'll see how things progress.

**Comment by keevie** [comment]
@buckett I wasn't planning on it, but am open to that if it makes sense! For now it seems like we'll be okay here.
